piq/functional/colour_conversion.py

```diff
@@ -54,7 +54,7 @@ def xyz2lab(x: torch.Tensor, illuminant: str = 'D50', observer: str = '2') -> to
     """
     epsilon = 0.008856
     kappa = 903.3
-    illuminants = \
+    illuminants: Dict = \
```

Comment:
Same here

Reply:
I'll update this to `Dict[str, Dict]` because otherwise `mypy` can't determine correct type for dict of dicts.